### DISCUSSION TITLE
Recover from recyclebin

### DISCUSSION THREAD

**Discussion by testmodem** [YetiForceCompany/YetiForceCRM#16481] [General] [answered]
Dear all 

After spending few hours I have failed to located any such option . Please advise.

1) How do I recover records from 'Recycle bin'. Where is 'Recycle bin' icon or anything located.

2) How do I undo 'Mass Archive' .

Thanks

**Comment by rskrzypczak** [comment, marked as answer]
If you want a recycle bin, you need to add it to the menu first.
![image](https://user-images.githubusercontent.com/10221999/190557908-e6cbd02c-fad1-47ab-a20e-d0d88a66f357.png)

However, the recycle bin is not required to manipulate the state of the records.
![bin](https://user-images.githubusercontent.com/10221999/190557733-be3c2a7a-d42a-4c8c-9c5b-98bb7bddc10e.gif)

**Comment by testmodem** [comment]
Thank you .. got it .. Yetiforce is so features rich .. amazing .

Thanks again
